source/fundamentals/builders/sort.txt

```diff
@@ -160,10 +152,7 @@ The following code snippet shows how to specify a descending sort on the
    
    collection.find().sort(descending("_id"));
 
-
-The code snippet above returns the documents in the
-:ref:`sort_example collection <sorts-builders-sort-example>`  
-in the following order: 
+The output of the example above should look something like this: 
```

Comment:
You can always tidy up a sentence by moving the subject around and using verbs to convey the other relevant information:
```suggestion
The above example should output something like this: 
```

Reply:
Thank you for the note. I agree that phrasing is much better. I think I got too into the habit of saying "above should look something like this". Will try to reuse text more carefully.